web_timeline/static/src/js/timeline_renderer.js

```diff
@@ -308,41 +308,164 @@ odoo.define('web_timeline.TimelineRenderer', function (require) {
         on_data_loaded: function (events, group_bys, adjust_window) {
             var self = this;
             var ids = _.pluck(events, "id");
-            return this._rpc({
-                model: this.modelName,
+            var group_by_field = self.view.fields[group_bys[0]];
+            if (group_by_field.type == 'one2many' | group_by_field.type == 'many2many') {
```

Comment:
@Tardo I'm not sure that it is necessary to check that the type are matching too when the value is the same in this case. Do you have a good reason to do that? thanks

Reply:
== and != are called the "evil twins" ... in this case you are right, doesn't need strict equality comparison. But it's a good practice to avoid possible type coercion.